### ISSUE TITLE
VBA Regional Office Information Architecture work

### ISSUE THREAD

**Issue by davidmpickett** [department-of-veterans-affairs/va.gov-cms#15490]
## User Story or Problem Statement
In [conjunction with CAIA](https://github.com/department-of-veterans-affairs/va.gov-team/issues/64934), determine path forward for the IA of VBA Regional Offices.

- H1s, breadcrumbs, urls
- [Facility naming schema](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/facilities/naming-schema#vba-benefits-facilities-naming-schema)
- Drupal section names (not Veteran-facing)

**Stress cases to consider:**

Some current example of official names of Regional Offices and Satellite Offices that may cause issues when converting to URLs / H1s /breadcrumbs
- [Honolulu VA Regional Benefit Office and Spark M. Matsunaga Department of Veterans Affairs Medical Center](https://www.va.gov/find-locations/facility/vba_459)
  - Long names are long
- [VA Regional Benefit Office at U.S. Embassy in Manila](https://www.va.gov/find-locations/facility/vba_358)
  - This is the only Regional Office that doesn't start have a [region name] before the [facility type]
- [Fort Harrison VA Regional Benefit Office and Fort Harrison VA Medical Center](https://www.va.gov/find-locations/facility/vba_436)
  - When a Regional Office is at a shared location, sometimes the [region name] is repeated in the [shared location name] 
  - See also: Anchorage VA Regional Benefit Office at Anchorage VA Medical Center
- [VA Regional Benefit Satellite Office at Gerald W. Heaney Federal Building & U.S. Courthouse](https://www.va.gov/find-locations/facility/vba_335d)
  - Punctuation in the shared location name
- [VA Regional Benefit Satellite Office at Samuel S. Stratton Department of Veterans Affairs Medical Center](https://www.va.gov/find-locations/facility/vba_306b)
  -  Long names are long (satellite office edition)

## Acceptance Criteria - Sprint 93 - 3 points
- [x] Met with @kristinoletmuskat
- [x] Created [PR to update VBA naming schema](https://github.com/department-of-veterans-affairs/va.gov-team/pull/66167) to reflect current truth
- [x] Identified stress cases for VBA facility names

## Acceptance Criteria - Sprint 94 - 1 point
- [x] Coordinated with @mmiddaugh and @kristinoletmuskat about next steps
- [x] Updated some related tickets
- [x] Merged PR

## Acceptance Criteria - Sprint 95 - estimate 3 point
- [x] Meeting on 10/11 with @mmiddaugh, @kristinoletmuskat, & @mnorthuis 
- [x] Created proposal for [VBA short names](https://dvagov.sharepoint.com/:x:/s/SitewideFacilitiesTeam/ESGDTL0pkqdDiKT3oqcFoZEBuEtSx0NdGZIbvUwY7TwQ9w?e=9amPaQ)
- [x] Shared short name proposal with Facilities team in [slack](https://dsva.slack.com/archives/C0FQSS30V/p1697117837065219)
- [x] Incorporate any updates or feedback into proposal

## Future sprint (or different ticket)
- [ ] Work with PM/PO takes short name proposal to VBA national stakeholders
- [ ] Once proposal is approved, update [Naming Schema](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/facilities/naming-schema#vba-benefits-facilities-naming-schema) and update any follow up tickets
- [ ] Submit Collab Cycle Analytics request per [46029](https://github.com/department-of-veterans-affairs/va.gov-team/issues/46029)

**Comment by davidmpickett** [comment]
@jilladams @xiongjaneg  
**End of sprint 94 update:** 
- sprint 94 ACs are complete
- ACs for Sprint 95 added
- Points can be updated at sprint boundary

**Comment by jilladams** [comment]
Accounted for in S94 reporting; moving back to Current sprint not started for S95.

**Comment by davidmpickett** [comment]
Still more work to do, but notes and examples from meeting today with @mmiddaugh @mnorthuis @kristinoletmuskat 

**URL structure**
- No on name-spacing to va.gov/benefit-offices/[facility-name] or similar
  - References an empty directory
  - Doesn't add value to Veteran / may increase cognitive load
  - Long-term we want to move away from Facility-centric IA towards service-centric IA, so shouldn't build enshrine facility logic in URL

**Suggestions from meeting for creating short names**
- We can **drop VA** from names because it's already in the URL / breadcrumbs
- **Satellite Offices should drop the word Regional**, reserve that for Regional Offices
- Shared location could be handled in an H2 ala [Prescott Vet Center](https://www.va.gov/prescott-vet-center/)
- _Might_ even be a case for dropping Satellite and Regional

**Possible examples**
- VA Regional Benefit Satellite Office at Amarillo VA Medical Center
  - Amarillo Benefit Office (at Amarillo VA Medical Center)
  - Amarillo Satellite Office (at Amarillo VA Medical Center)
  - Amarillo Satellite Benefit Office (at Amarillo VA Medical Center)
- Cleveland VA Regional Benefit Office
  - Cleveland Regional Office
  - Cleveland Benefit Office
  - Cleveland Regional Benefit Office 
- VA Regional Benefit Satellite Office at Captain James A. Lovell Federal Health Care Center
  - Lovell Satellite Office (at Captain James A. Lovell Federal Health Care Center)

**Next steps**
- Dave to create proposed short name convention for review/discussion
- Short names would drive URLs, breadcrumbs, h1s

**Comment by davidmpickett** [comment]
First pass at short name schema
[vba_short_names.xlsx](https://github.com/department-of-veterans-affairs/va.gov-cms/files/12876062/vba_short_names.xlsx)

**Comment by davidmpickett** [comment]
Moved over to Sharepoint for comments and collaboration: [VBA short names proposal](https://dvagov.sharepoint.com/:x:/s/SitewideFacilitiesTeam/ESGDTL0pkqdDiKT3oqcFoZEBuEtSx0NdGZIbvUwY7TwQ9w?e=9amPaQ)

**Comment by laflannery** [comment]
Noting a correction in the comment above - The pattern in Prescott Vet center is not an H1 and H2. It's an H1 and associated text

**Comment by davidmpickett** [comment]
Updated proposal export 
[VBA Services trimmed.xlsx](https://github.com/department-of-veterans-affairs/va.gov-cms/files/13044836/VBA.Services.trimmed.xlsx)

**Comment by davidmpickett** [comment]
Update from UX refinement - #15660 might also be good to do before taking a proposal to VBA

**Comment by davidmpickett** [comment]
@xiongjaneg @jilladams 
Calling this complete for S95. For the ACs under "Future sprint (or different ticket)" might make sense to move those ACs to a separate ticket since that's Product work anyway (and because #15660 might block the product work)

**Comment by xiongjaneg** [comment]
Moved future ACs to 15752 and closing
